Security flaws check improvements (#4393)
wazuh/wazuh-qa: tests/scans/code_analysis/README.md

- If the test passes, no actions are needed, everything is correct.

- If the test fails, new code vulnerabilities will be found in `wazuh-qa/tests/scans/code_analysis/new_flaws_{module}.json`.
  - We analyze the new vulnerabilities found in the module and report them in GitHub issues.
  - We move the vulnerabilities to the `to_fix` key of the known flaws JSON file.
  - If the new vulnerability is considered a false positive, we add it to the `false_positives` list of the dictionary in its respective `known_flaws` JSON file.
  - If the new vulnerability is a real vulnerability, we solve the problem reported and remove the flaw from the known flaws file.

> The test also updates the known_flaws files automatically. If we have a look to a known_flaws file, we will see that each flaw dictionary contains information like the line number or range. This information si the one updated by the test. The test also removes flaws from the known_flaws file if they don't appear in the Bandit output.

Reviewer: ```suggestion
The test also updates the known_flaws files automatically. If we have a look at a known_flaws file, we will see that each flaw dictionary contains information like the line number or range. This information is the one updated by the test. The test also removes flaws from the known_flaws file if they don't appear in the Bandit output.
```
